src/interface/desktop/chat.html

```diff
@@ -675,28 +675,35 @@
             await loadChat();
         });
 
-        async function loadChat() {
-            // Load chat history
+        async function getChatHistoryUrl() {
             const hostURL = await window.hostURLAPI.getURL();
             const khojToken = await window.tokenAPI.getToken();
             const headers = { 'Authorization': `Bearer ${khojToken}` };
 
-            let firstRunSetupMessageRendered = false;
             let chatBody = document.getElementById("chat-body");
             chatBody.innerHTML = "";
             let chatHistoryUrl = `${hostURL}/api/chat/history?client=desktop`;
             if (chatBody.dataset.conversationId) {
                 chatHistoryUrl += `&conversation_id=${chatBody.dataset.conversationId}`;
             }
+            return { chatHistoryUrl, headers };
+        }
+
+        async function loadChat() {
+            // Load chat history and body
+            const hostURL = await window.hostURLAPI.getURL();
```

Comment:
This can be nixed.

Reply:
`hostURL` is used in `loadChat`function futher down